cmd/links.go

```diff
@@ -0,0 +1,132 @@
+	linksUpdateLongDescription = `Use this command to update all linked files references inside the current directory.`
+	linksListLongDescription   = `Use this command to list all packages that have linked file references that include the current directory.`
+)
+
+func setupLinksCommand() *cobraext.Command {
+	cmd := &cobra.Command{
+		Use:   "links",
+		Short: "Manage linked files",
+		Long:  linksLongDescription,
+		RunE: func(parent *cobra.Command, args []string) error {
+			return cobraext.ComposeCommandsParentContext(parent, args, parent.Commands()...)
+		},
+	}
+
+	cmd.AddCommand(getLinksCheckCommand())
+	cmd.AddCommand(getLinksUpdateCommand())
+	cmd.AddCommand(getLinksListCommand())
+
+	return cobraext.NewCommand(cmd, cobraext.ContextBoth)
+}
+
+func getLinksCheckCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "check",
+		Short: "Check for linked files changes",
+		Long:  linksCheckLongDescription,
+		Args:  cobra.NoArgs,
+		RunE:  linksCheckCommandAction,
+	}
+	return cmd
+}
+
+func linksCheckCommandAction(cmd *cobra.Command, args []string) error {
+	cmd.Printf("Check for linked files changes\n")
+	pwd, err := os.Getwd()
+	if err != nil {
+		return fmt.Errorf("reading current working directory failed: %w", err)
+	}
+
+	linkedFiles, err := builder.AreLinkedFilesUpToDate(pwd)
+	if err != nil {
+		return fmt.Errorf("checking linked files are up-to-date failed: %w", err)
+	}
+	for _, f := range linkedFiles {
+		if !f.UpToDate {
+			cmd.Printf("%s is outdated.\n", f.LinkFilePath)
+		}
+	}
+	if len(linkedFiles) > 0 {
+		return fmt.Errorf("linked files are outdated")
+	}
+	return nil
+}
+
+func getLinksUpdateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update",
+		Short: "Update linked files checksums if needed.",
+		Long:  linksUpdateLongDescription,
+		Args:  cobra.NoArgs,
+		RunE:  linksUpdateCommandAction,
+	}
+	return cmd
+}
+
+func linksUpdateCommandAction(cmd *cobra.Command, args []string) error {
+	cmd.Printf("Update linked files checksums if needed.\n")
+	pwd, err := os.Getwd()
+	if err != nil {
+		return fmt.Errorf("reading current working directory failed: %w", err)
+	}
+
+	linkedFiles, err := builder.UpdateLinkedFilesChecksums(pwd)
+	if err != nil {
+		return fmt.Errorf("updating linked files checksums failed: %w", err)
+	}
+
+	for _, f := range linkedFiles {
+		if !f.UpToDate {
+			cmd.Printf("%s is outdated.\n", f.LinkFilePath)
+		}
+	}
+
+	return nil
+}
+
+func getLinksListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List packages linking files from this path",
+		Args:  cobra.NoArgs,
+		RunE:  linksListCommandAction,
+	}
+	return cmd
+}
+
+func linksListCommandAction(cmd *cobra.Command, args []string) error {
+	pwd, err := os.Getwd()
+	if err != nil {
+		return fmt.Errorf("reading current working directory failed: %w", err)
+	}
+
+	packages, err := builder.ListPackagesWithLinkedFilesFrom(pwd)
```

Comment:
I wonder if it wouldn't be better to list the files instead of the packages. Advantages:
- It makes easier to identify referenced link files.
- This makes the command less dependant of packages.
- It is easier to find the packages once you have the files.

Or maybe we can have a flag in this command to select if the files or the packages are wanted, but I would default to the files.